torch/csrc/jit/ir/node_hashing.cpp

```diff
@@ -213,6 +213,10 @@ size_t HashNode::operator()(const Node* k) const {
         type->isSubtypeOf(NumberType::get()) &&
         k->kindOf(attr::value) == AttributeKind::f) {
       constant_hash = std::hash<double>{}(k->f(attr::value));
+    } else if (
```

Comment:
updating `()` operator here helps access elements from the unordered_map `c_constants`